### ISSUE TITLE
Cannot include build 'build-logic' in build 'module_name_here'. This is not supported yet.

### ISSUE THREAD

**Issue by jakoss** [PatilShreyas/compose-report-to-html#27]
I'm trying to run the gradle plugin on some of my modules. Since we are using gradle conventions (which are in the gradle project named build-logic) i applied plugin globally in root build.gradle like this: 
```
alias libs.plugins.composeCompilerReportGenerator apply false
```

And then i apply the plugin only in selected modules via convention file, like this:
```
plugins {
    id 'dev.shreyaspatil.compose-compiler-report-generator'
}
```

When i do that i get the error:
```
Cannot include build 'build-logic' in build 'module_name_here'. This is not supported yet.
```

When i apply plugin directly on a module i want to generate reports on i have the same exact issue. For some reason this plugin does not like composite builds

**Comment by PatilShreyas** [comment]
@jakoss I tried it on my end and this issue is not occurring. Can you please check again? Or if possible, can you provide a sample project to reproduce this?

**Comment by jakoss** [comment]
Sure thing, [here's the branch](https://github.com/jakoss/AndroidArchitectureTemplate/tree/feature/compose-compiler-html-reports). Try and run `./gradlew :core:ui:releaseComposeCompilerHtmlReport` on this branch.

I had to turn off configuration cache because it still doesn't work with the plugin (you can enable it by uncommenting last 2 entries in [gradle.properties](https://github.com/jakoss/AndroidArchitectureTemplate/blob/feature/compose-compiler-html-reports/gradle.properties)

**Comment by PatilShreyas** [comment]
I haven't yet cloned your repo, but I can see you're using Gradle Wrapper of 8.0, and this plugin is built with 7.6. My assumption is maybe you're using some new features in your project. I'll dig more into this. Thanks for raising this!

**Comment by PatilShreyas** [comment]
Just came across this open issue: https://github.com/gradle/gradle/issues/8246

In this plugin, we are using `GradleBuild` task which is not working with composite builds. Have to find workaround for this.

**Comment by PatilShreyas** [comment]
Can see that the `GradleConnector` tooling API can fix this. If it works fine, will replace this whole implementation from root.
Ref: https://github.com/researchgate/gradle-release/issues/304#issuecomment-1083692649

**Comment by PatilShreyas** [comment]
Thanks, @jakoss for raising this. This has been fixed and released in `v1.0.0`
